cmd/root.go

```diff
@@ -117,13 +119,21 @@ func Execute() {
 func getConfigHome(opts *rootOptions) (string, error) {
 	if opts.configHome != "" {
 		output.Debugf("Using config directory from flag: %s", opts.configHome)
-		return opts.configHome, nil
+		absHome, err := filepath.Abs(opts.configHome)
```

Comment:
this is repeated. Should it be a nested helper function something like.

```
getAbsPath := func(path, description string) (string, error) {
		absPath, err := filepath.Abs(path)
		if err != nil {
			return "", fmt.Errorf("failed to get absolute path for %s: %w", description, err)
		}
		return absPath, nil
	}
```

Reply:
I feel like this level of repetition is just annoyingly inherent to golang (and trying to reduce it would just complicate the code). 

The only part we're really repeating is handling the error, and I think I prefer to more clearly distinguish the message between "we couldn't figure out the path you passed via `--context`" vs the more subtle "`KITOPS_HOME` is set somewhere and that's messing things up."

e.g.
```go
getAbsPath := func(...)

absHome, err := getAbsPath(opts.configHome, "--config parameter")
if err != nil {
    return err
}
```

vs.

```go
absHome, err := filepath.Abs(path)
if err != nil {
    return fmt.Errorf("<message>")
}
```